data/mainnet/tokens.js

```diff
@@ -45,6 +45,109 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['FARM', 'WETH'],
   },
+  note: {
+    chain: CHAINS_ID.ETH_MAINNET,
+    logoUrl: './icons/note.png',
+    tokenAddress: addresses.NOTE,
+    decimals: '8',
+    vaultAddress: null,
+    priceFunction: {
+      type: GET_PRICE_TYPES.COINGECKO_ID,
+      params: ['notional-finance'],
+    },
+  },
+  notional_DAI: {
```

Comment:
Please add `isNew: true` to all the new vaults, this will give them the "New" icon on the UI.